jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java

```diff
@@ -331,6 +339,11 @@ private void executionQueue(Execution message) {
 
 
     private void workerTaskResultQueue(WorkerTaskResult message) {
+        if (skipExecutionService.skipExecution(message.getTaskRun().getTaskId())) {
```

Comment:
Same here on the executor side please

Reply:
Here, the ExecutorSerice is not used so there is no choice but to do it like this.

Reply:
you have an option [here](https://github.com/kestra-io/kestra/blob/ba1eaa309ce37fd91cd53b17a60d9639d16f148d/jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java#L357)